spec/ui/home_page_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+describe Pages::Home do
+  context 'Dfe Homepage' do
+    before(:each) do
+      @home_page = Pages::Home.new
+    end
+
+    it 'Should be displayed when the user navigates to the hame page and clicks the header logo' do
+      @home_page.load
+      @home_page.header.logo.click
+
+      expect(@home_page).to be_displayed
+    end
+
+    it 'should navigate away from the home page when the user clicks sign in' do
+      @home_page.load
+      @home_page.header.sign_in.click
+      @sign_in = Pages::SignIn.new
```

Comment:
Again - I think `sign_in` should be define via `let` outside the `it` block.

Reply:
I'll change to the let and pass the block.
In a short time, this will need to change again as the API for the application changes so far as a domain perspective,